jake/pip/pip.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, targets=None, requirements_file_path=None):
     self.requirements_file_path = requirements_file_path
     if targets:
       self._working_set = pkg_resources.WorkingSet(ast.literal_eval(targets))
-    self._coords = self.generate_dependencies()
+    self.__coords = self.generate_dependencies()
```

Comment:
Did this sneak back in?

Reply:
Would you please correct it by yourself?
If i do, the CI will need another commit from you in order to run.

Reply:
Perfect. I didn't want to muck with your PR without consent. Duck and cover, here it comes.